D1673/blas_interface.md

```diff
@@ -878,6 +889,11 @@ layouts.  This includes low-rank updates to symmetric or Hermitian
 matrices, and matrix-matrix multiplication with symmetric or Hermitian
 matrices.
 
+It is an open question whether our algorithms should also allow any 
```

Comment:
Remove this and fix wording above that excludes unique layouts not in this proposal.  Functions already say the right thing about layouts.

Reply:
crtrott: So we can replace the first bullet point with any layout for which always unique is true